.github/workflows/automerge.yml

```diff
@@ -0,0 +1,43 @@
+name: Devin.ai Auto Merge
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    # Only allow Devin.ai GitHub App to trigger auto-merge
+    if: github.event.pull_request.user.login == 'devin-ai-integration[bot]'
```

Comment:
This is probably the only change required at https://github.com/ThePrimeagen/chussy/blob/7b6a597229b86b122989327309784258ea6345c8/.github/workflows/auto-merge.yml#L18-L20